packages/autorest.python/autorest/codegen/models/code_model.py

```diff
@@ -77,6 +77,14 @@ def __init__(
             t for t in self.types_map.values() if isinstance(t, CombinedType) and t.name
         ]
 
+    @property
+    def enable_flatten(self) -> bool:
```

Comment:
no need for this property we can just use `self.options["models_mode"]`.

Reply:
done